projects/packages/wp-js-data-sync/.phpcs.dir.xml

```diff
@@ -27,35 +27,30 @@
 		</properties>
 	</rule>
 
-	<rule ref="MediaWiki.Usage.ForbiddenFunctions.isset">
-		<severity>0</severity>
+	<rule ref="MediaWiki.Usage.ForbiddenFunctions">
+		<exclude name="MediaWiki.Usage.ForbiddenFunctions.isset"/>
 	</rule>
 
-	<rule ref="Squiz.Commenting.FunctionComment.MissingParamComment">
-		<severity>0</severity>
+	<rule ref="Squiz.Commenting.FunctionComment">
+		<exclude name="Squiz.Commenting.FunctionComment.MissingParamComment"/>
+		<exclude name="Squiz.Commenting.FunctionComment.MissingParamName"/>
+		<exclude name="Squiz.Commenting.FunctionComment.MissingParamTag"/>
+		<exclude name="Squiz.Commenting.FunctionComment.MissingReturn"/>
+		<exclude name="Squiz.Commenting.FunctionComment.ParamCommentFullStop"/>
 	</rule>
-	<rule ref="Squiz.Commenting.FunctionComment.MissingParamName">
-		<severity>0</severity>
+	<rule ref="Generic.Commenting.DocComment">
+		<exclude name="Generic.Commenting.DocComment.MissingShort"/>
 	</rule>
-	<rule ref="Squiz.Commenting.FunctionComment.MissingParamTag">
-		<severity>0</severity>
+	<rule ref="Squiz.Commenting.FunctionComment">
+		<exclude name="Squiz.Commenting.FunctionComment.Missing"/>
```

Comment:
This should be nested in the above group (line 34).

Reply:
🤷 Phpcs doesn't seem to mind, and the original seemed to intentionally have all the "Missing" sniffs listed as a group.